BrowserKit/Sources/ToolbarKit/TabNumberButton.swift

```diff
@@ -0,0 +1,68 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+
+final class TabNumberButton: ToolbarButton {
+    // MARK: - UX Constants
+    struct UX {
+        static let cornerRadius: CGFloat = 2
+        static let titleFont = UIFont.boldSystemFont(ofSize: 11)
+
+        // Tab count related constants
+        static let defaultCountLabelText = "0"
+        static let maxTabCountToShowInfinity = 100
```

Comment:
According to the logic later in code the last number displayed is 99. I suggest to change the constant here and adjust the logic accordingly.